client/python/gradio_client/client.py

```diff
@@ -473,13 +479,25 @@ def _render_endpoints_info(
         human_info += "    Parameters:\n"
         if endpoints_info["parameters"]:
             for info in endpoints_info["parameters"]:
-                human_info += f"     - [{info['component']}] {utils.sanitize_parameter_names(info['label'])}: {info['type_python']} ({info['type_description']})\n"
+                desc = (
+                    f" ({info['python_type']['description']})"
+                    if info["python_type"].get("description")
+                    else ""
```

Comment:
Nit: maybe we can always include a `description` key, and it can just be "" by default. That way we'll have a more consistent scheme and don't have to check for the presence of the `description` key

Reply:
Actually I think that's what I'm [doing](https://github.com/gradio-app/gradio/pull/4039/files#diff-d181109e25b5f0829054b7774d79faf8fdff3eba38069c63fcf0651a9b84d0aeR519) 

`description` is always present. "" and None are both false so this is checking whether or the the `description` is empty. I think the use of `get` is confusing here but imo calling `get` is better practice. Will think of a way to rewrite this that's clearer!

Reply:
Ah you are! So then why do you need the if/else here? Sorry I'm probably missing something but wouldn't 
```
f" ({info['python_type']['description']})"
```
be equivalent?

Reply:
If the description is empty string, then `view_api()` would show empty parentheses, e.g. `parameter3: int | float ()`

Reply:
Oh sorry you're totally right. Scratch this